cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -187,6 +194,13 @@ instance FromJSON PartialNodeConfiguration where
       pncTargetNumberOfEstablishedPeers <- Last <$> v .:? "TargetNumberOfEstablishedPeers"
       pncTargetNumberOfActivePeers      <- Last <$> v .:? "TargetNumberOfActivePeers"
 
+      -- Enable P2P switch
+      p2pSwitch <- v .:? "EnableP2P" .!= Just False
```

Comment:
Why not:
```
p2pSwitch <- v .:? "EnableP2P" .!= Just DisabledP2PMode
```

Reply:
Just for the sake of UX: In the config file the user should just need to write `"EnableP2P" : true`, instead of something else 🙂